packages/angular/cli/commands/definitions.json

```diff
@@ -57,12 +57,12 @@
         "interactive": {
           "type": "boolean",
           "default": "true",
-          "description": "When false, disables interactive input prompts."
+          "description": "Enable interactive input prompts."
```

Comment:
nit: maybe be consistent on the verb form used (e.g. "enable", but "disable**s**" for the next option)?

Reply:
Agree